src/builder.rs

```diff
@@ -32,20 +41,22 @@
 //! ```
 
 use hal;
+use hal::digital::OutputPin;
 
 use crate::displayrotation::DisplayRotation;
 use crate::displaysize::DisplaySize;
-use crate::interface::I2cInterface;
+use crate::interface::{I2cInterface, SpiInterface};
 use crate::mode::displaymode::DisplayMode;
 use crate::mode::raw::RawMode;
 use crate::properties::DisplayProperties;
 
 /// Builder struct. Driver options and interface are set using its methods.
 #[derive(Clone, Copy)]
-pub struct Builder {
+pub struct Builder<CS=NoOutputPin> {
```

Comment:
I haven't seen the `=` syntax before. What does it do/what should I Google for?

Reply:
It's a [defaulted type parameter](https://github.com/nox/rust-rfcs/blob/master/text/0213-defaulted-type-params.md). This way the user doesn't have to specify a type if they don't need that pin.

Reply:
That's awesome. Rust never ceases to amaze!